rocAL/rocAL_pybind/amd/rocal/types.py

```diff
@@ -18,71 +18,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-
-
-# RocalStatus
-from rocal_pybind.types import OK
-from rocal_pybind.types import CONTEXT_INVALID
-from rocal_pybind.types import RUNTIME_ERROR
-from rocal_pybind.types import UPDATE_PARAMETER_FAILED
-from rocal_pybind.types import INVALID_PARAMETER_TYPE
-
-#  RocalProcessMode
-from rocal_pybind.types import GPU
-from rocal_pybind.types import CPU
-
-#  RocalTensorOutputType
-from rocal_pybind.types import UINT8
-from rocal_pybind.types import FLOAT
-from rocal_pybind.types import FLOAT16
-from rocal_pybind.types import UINT8
-
-#  RocalOutputMemType
-from rocal_pybind.types import CPU_MEMORY
-from rocal_pybind.types import GPU_MEMORY
-from rocal_pybind.types import PINNED_MEMORY
-
-# RocalImageSizeEvaluationPolicy
-from rocal_pybind.types import MAX_SIZE
-from rocal_pybind.types import USER_GIVEN_SIZE
-from rocal_pybind.types import MOST_FREQUENT_SIZE
-from rocal_pybind.types import MAX_SIZE_ORIG
-from rocal_pybind.types import USER_GIVEN_SIZE_ORIG
-
-#      RocalImageColor
-from rocal_pybind.types import RGB
-from rocal_pybind.types import BGR
-from rocal_pybind.types import GRAY
-from rocal_pybind.types import RGB_PLANAR
-
-#     RocalTensorLayout
-from rocal_pybind.types import NHWC
-from rocal_pybind.types import NCHW
-
-#     RocalDecodeDevice
-from rocal_pybind.types import HARDWARE_DECODE
-from rocal_pybind.types import SOFTWARE_DECODE
-
-#     RocalDecodeDevice
-from rocal_pybind.types import DECODER_TJPEG
-from rocal_pybind.types import DECODER_OPENCV
-from rocal_pybind.types import DECODER_HW_JEPG
-from rocal_pybind.types import DECODER_VIDEO_FFMPEG_SW
-from rocal_pybind.types import DECODER_VIDEO_FFMPEG_HW
-
-#     RocalResizeScalingMode
-from rocal_pybind.types import SCALING_MODE_DEFAULT
-from rocal_pybind.types import SCALING_MODE_STRETCH
-from rocal_pybind.types import SCALING_MODE_NOT_SMALLER
-from rocal_pybind.types import SCALING_MODE_NOT_LARGER
-
-#     RocalResizeInterpolationType
-from rocal_pybind.types import NEAREST_NEIGHBOR_INTERPOLATION
-from rocal_pybind.types import LINEAR_INTERPOLATION
-from rocal_pybind.types import CUBIC_INTERPOLATION
-from rocal_pybind.types import LANCZOS_INTERPOLATION
-from rocal_pybind.types import GAUSSIAN_INTERPOLATION
-from rocal_pybind.types import TRIANGULAR_INTERPOLATION
+from rocal_pybind.types import *
```

Comment:
@SundarRajan28 * import will give codacy errors. Please revert back

Reply:
Removed the * import and reverted back to the manual imports